docs/config.rst

```diff
@@ -409,6 +409,11 @@ Run
    Commands to run before running the :ref:`commands`. All evaluation and configuration logic applies from
    :ref:`commands`.
 
+   .. note::
+
+     When you use the ``#`` sign inside ``commands_pre``, you need to escape it, e.g.
```

Comment:
Please remove these from here. These config flags contain the generic description of the configuration. This behavior is also not bound just to commands. This is a generic ini config behaviour and should be documented in a ini config description area.